ImplicitPackageReferenceUnitTests/ImplicitPackageReferenceUnitTests.cs

```diff
@@ -124,86 +148,55 @@ public void WhenTaskIsPassedXMLFileTaskReturnsFalse()
             var implicitPacker = new AddImplicitPackageReferences(log);
             TaskItem implicitDependency = new TaskItem("Microsoft.AspNetCore.Http.Abstractions");
             implicitPacker.AssetsFilePath = "WrongFile.xml";
-            implicitPacker.DependenciesToVersionAndPackage = new TaskItem[] { implicitDependency };
+            implicitPacker.ImplicitPackageReferences = new TaskItem[] { implicitDependency };
             Assert.IsFalse(implicitPacker.Execute());
             Assert.AreEqual("AddImplicitPackageReferences failed, could not parse file WrongFile.xml. Exception:Unexpected character encountered while parsing value: <. Path '', line 0, position 0.", log.ErrorMessage);
         }
 
-        [TestMethod]
-        [DeploymentItem(@"MissingLibrary.json")]
-        public void WhenJsonFileIsMissingLibrarySectionTaskReturnsFalse()
```

Comment:
Removed this test because A. It doesn't make much sense (when would this actually happen in practice), and B. because the library section no longer matters.